sklearn/model_selection/tests/test_validation.py

```diff
@@ -1259,7 +1339,8 @@ def test_cross_val_predict_class_subset():
         assert_array_almost_equal(expected_predictions, predictions)
 
         # Testing unordered labels
-        y = [1, 1, -4, 6]
+        y = np.array([x//20 for x in range(-100, 100, 2)])
+        y = shuffle(y, random_state=0)
```

Comment:
or just `y = np.repeat([1, 1, -4, 6], 25)`. Always a good idea to avoid unnecessary compilcations ...